blocks/subscriptions-block/components/OfferToProductList/index.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import useSales from "../useSales";
+import GridList from "../GridList";
+import OfferCard from "../OfferCard";
+
+const OfferToProductList = ({ offer, isLoggedIn, checkoutURL, loginURL, className }) => {
+	const { Sales } = useSales();
+
+	const buildOffers = () => {
+		const offers = [];
+		let productIdx;
+		let strategiesIdx;
+		if (offer.products) {
+			const { products } = offer;
+			for (productIdx = 0; productIdx < products.length; productIdx += 1) {
+				const { pricingStrategies: strategies } = products[productIdx];
+				for (strategiesIdx = 0; strategiesIdx < strategies.length; strategiesIdx += 1) {
+					const features =
+						typeof products[productIdx].attributes !== "undefined" &&
+						products[productIdx].attributes.length !== 0
+							? products[productIdx].attributes.map((feature) => ({
+									featureText: feature.value,
+							  }))
+							: [];
+					const { sku } = products[productIdx];
+					const { priceCode } = strategies[strategiesIdx];
+					offers.push({
+						key: strategies[strategiesIdx].pricingStrategyId,
+						subHeadline: strategies[strategiesIdx].description,
+						headline: strategies[strategiesIdx].name,
+						actionText: strategies[strategiesIdx].summary,
+						actionEvent: () => {
+							Sales.clearCart()
+								.then(() =>
+									Sales.addItemToCart([
+										{
+											sku,
+											priceCode,
+											quantity: 1,
+										},
+									])
+								)
+								.then(() => {
+									if (isLoggedIn) {
+										window.location.href = checkoutURL;
+										return;
+									}
+									window.location.href = `${loginURL}?redirect=${checkoutURL}`;
+								});
+						},
+						features,
+					});
+				}
+			}
+		}
+		return offers;
+	};
+
+	const builtOffers = buildOffers();
+
+	return (
+		<GridList className={className}>
```

Comment:
There's a `Grid` component in `arc-themes-components`, could that be used here instead of making this new component?

Reply:
@vgalatro Good call, now I'm using Grid component